gnmi_server/server.go

```diff
@@ -503,6 +530,7 @@ func (s *Server) Set(ctx context.Context, req *gnmipb.SetRequest) (*gnmipb.SetRe
 		common_utils.IncCounter(common_utils.GNMI_SET_FAIL)
 	}
 
+	s.SaveStartupConfig()
```

Comment:
No locks in both Set() and SaveOnSetEnabled(). Is it safe to allow parallel db writes and config saves?